### ISSUE TITLE
Unexpected error fetching weatherflow data: 'wind_avg'

### ISSUE THREAD

**Issue by dhover** [briis/hass-weatherflow#6]
I got an error with the new integration.

```
2021-12-04 22:31:36 ERROR (MainThread) [custom_components.weatherflow] Unexpected error fetching weatherflow data: 'wind_avg'
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 187, in _async_refresh
    self.data = await self._async_update_data()
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 147, in _async_update_data
    return await self.update_method()
  File "/config/custom_components/weatherflow/__init__.py", line 104, in async_update_data
    data: ObservationDescription = await weatherflowapi.update_observations()
  File "/usr/local/lib/python3.9/site-packages/pyweatherflowrest/api.py", line 185, in update_observations
    beaufort: BeaufortDescription = self.calc.beaufort(obervations["wind_avg"])
KeyError: 'wind_avg'

```

**Comment by briis** [comment]
Strange error. This is a standard field in the JSON reply, and I am not getting this. Anyway, I will release a new version tomorrow, so let us see if this fixes it.

**Comment by briis** [comment]
Hopefully fixed  in 0.1.9 release.
